examples/tangent_pca_s2.py

```diff
@@ -38,10 +40,10 @@ def main():
     geodesic_points_0 = geodesic_0(t)
     geodesic_points_1 = geodesic_1(t)
 
-    print(
+    logging.info(
         'Coordinates of the Log of the first 5 data points at the mean, '
         'projected on the principal components:')
-    print(tangent_projected_data[:5])
+    logging.info('\n{}'.format(tangent_projected_data[:5]))
```

Comment:
Is \n needed here?

Reply:
For this case I was looking to align all 5 data points since the first entry is shifted to the right due to the addition of the message logging level. I did this just for aesthetic reasons and is not required. Without the newline, the message will appear as shown below .
![q2](https://user-images.githubusercontent.com/56150991/75584140-efb20d00-5a23-11ea-84ca-cb33e03619b1.png)

Reply:
Oh I get it, great!